cs251tk/toolkit/args.py

```diff
@@ -165,4 +168,62 @@ def process_args():
     assignments = get_assignments_from_args(**args)
     stogit = compute_stogit_url(**args, _now=datetime.date.today())
 
+    print_args(args)
+    print_students(students)
+    print_assignments(assignments)
+    debug("stogit URL: " + stogit)
+
     return args, students, assignments, stogit
+
+
+def print_args(args):
+    debug("Command Line Arguments:")
+    debug("input_items:             " + str(args['input_items']))
```

Comment:
Could we just let Python format these, and just print `args`?
```suggestion
    debug(args)
```